blockchain/blockchain.go

```diff
@@ -407,15 +467,6 @@ func (bc *BlockChain) loadLastState() error {
 		logger.Error("Head block missing, resetting chain", "hash", head.String())
 		return bc.Reset()
 	}
-	// Make sure the state associated with the block is available
```

Comment:
Is it okay to remove this? It seems we have several program points calling loadLastState().

Reply:
Yes, these code fragments are pulled out of `loadLastState()` and all callers are updated.